src/components/routes/Home.js

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+import HomeIcon from '@mui/icons-material/Home';
+import SettingsIcon from '@mui/icons-material/Settings';
+import { useSelector, useDispatch } from 'react-redux';
+import getCountries from 'features/FetchAPI';
+import CountriesList from '../CountriesList';
+
+const Homepage = () => {
+  const dispatch = useDispatch();
+  const countries = useSelector((state) => state.countries.countries);
+
+  const [searchItem, setSearchItem] = useState('');
+
+  useEffect(() => {
+    if (countries.length === 0) {
+      dispatch(getCountries());
+    }
+  }, [dispatch, countries.length]);
+
+  const newCountries = countries.filter(
+    (country) => country.name.toLowerCase().includes(searchItem.toLowerCase())
+      || country.region.toLowerCase().includes(searchItem.toLowerCase()),
+  );
+
+  const handleSearch = (e) => {
+    e.preventDefault();
+    setSearchItem(e.target.value);
+  };
+
+  return (
+    <div>
+      <header className="home-header">
+        <div className="logo">
+          <HomeIcon className="svg_icons" />
+        </div>
+        <div className="search">
+          {' '}
+          <input
+            type="text"
+            name="searchItem"
+            placeholder="search for a country"
+            value={searchItem}
+            onChange={handleSearch}
```

Comment:
Good work creating a search bar 💯 , but I believe that you misunderstand the requirements of this project

There's no need to create a search bar, the requirements say that you need to filter the list of objects from your API by some property, like `name` `Population` `Continent` `Currency`, and other applicable properties from the objects

- Kindly create the necessary filter to list items by name or another property from your objects

Reply:
Hello @BregornOriginal, Thanks for the review. On the filter issue, I have filtered using the property `name` as per the requirement. Kindly check lines 20 to 23.